app/components/dashboard/diabetes/downloads_dropdown_component.html.erb

```diff
@@ -0,0 +1,72 @@
+<div class="dropdown show mb-24px bs-small mb-lg-0">
+  <a class="btn btn-sm dropdown-toggle c-black bg-white" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <i class="fa fa-arrow-circle-down w-16px ta-center mr-4px c-blue"></i>
+    Downloads (DM)
+  </a>
+
+  <div class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenuLink">
+
+    <!--    TODO: Remove facility from the array when patient list is available-->
+    <% unless region.region_type.in?(%w[block facility]) %>
+      <h6 class="dropdown-header c-grey-dark">
+        CSV files
+      </h6>
+    <% end %>
+
+    <% if region.district_region? && current_admin.feature_enabled?(:monthly_district_report)%>
+      <%= link_to(monthly_district_report_path, class: "dropdown-item") do %>
+        <i class="fa-regular fa-file-spreadsheet w-16px ta-center mr-8px c-blue"></i>
+        Monthly district report
+      <% end %>
+    <% end %>
+
+    <% if region.district_region? && current_admin.feature_enabled?(:monthly_district_data_download) %>
+      <%= link_to(monthly_district_data_path, class: "dropdown-item") do %>
+        <i class="fa-regular fa-file-spreadsheet w-16px ta-center mr-8px c-blue"></i>
+        Monthly <%= I18n.t("region_type.facility") %> data
+      <% end %>
+    <% end %>
+
+    <% if region.state_region? && current_admin.feature_enabled?(:monthly_state_data_download) %>
+      <%= link_to(monthly_state_data_path, class: "dropdown-item") do %>
+        <i class="fa-regular fa-file-spreadsheet w-16px ta-center mr-8px c-blue"></i>
+        Monthly <%= region.localized_child_region_type %> data
+      <% end %>
+    <% end %>
+
+    <% if region.region_type.in?(["district", "facility"]) %>
+      <% if current_admin.can_access?(region.source, :view_pii) %>
+        <%= link_to(patient_line_list_path, class: "dropdown-item") do %>
+          <i class="fa-regular fa-list-ul w-16px ta-center mr-8px c-blue"></i>
+          Patient line list
+        <% end %>
+      <% end %>
+    <% end %>
+
+    <% unless region.region_type.in?(%w[block]) %>
+     <div class="dropdown-divider"></div>
+    <% end %>
+
+    <h6 class="dropdown-header c-grey-dark">
+      <%= "#{quarter_string(Date.current.prev_quarter)}" %> Snapshots
```

Comment:
We should hide this section since it's not implemented yet. 

Also, we should check if we need to implement this section at all.